quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -53,6 +53,8 @@ pub struct Limits {
     pub(crate) unidirectional_data_window: InitialMaxStreamDataUni,
     pub(crate) max_open_bidirectional_streams: InitialMaxStreamsBidi,
     pub(crate) max_open_local_unidirectional_streams: stream::limits::LocalUnidirectional,
+    pub(crate) max_open_local_bidirectional_streams: Option<stream::limits::LocalBidirectional>,
+    pub(crate) max_open_remote_bidirectional_streams: Option<InitialMaxStreamsBidi>,
```

Comment:
Do these need to be optional?

Reply:
I don't think so. It makes things more complicated if it is.